resolwe/process/fields.py

```diff
@@ -1,7 +1,7 @@
 """Process input or output fields."""
 import collections
 import json
-
+from pathlib import Path
 import resolwe_runtime_utils
```

Comment:
I will remove it as soon as I decide how to properly include communicator in python processes. 
Current implementation does not feel right and I would really like to do it nicer since python processes must be future proof.